app/javascript/legacy/campaigner_facing/page_edit_bar.js

```diff
@@ -157,11 +155,13 @@ let PageEditBar = Backbone.View.extend({
     );
     if (data.status == 422) {
       ErrorDisplay.show(e, data);
-      $('.page-edit-bar__error-message').text(I18n.t('pages.edit.user_error'));
+      $('.page-edit-bar__error-message').text(
+        window.I18n.t('pages.edit.user_error')
+      );
       ee.emit('page:errors');
     } else {
       $('.page-edit-bar__error-message').text(
-        I18n.t('pages.edit.unknown_error')
+        window.I18n.t('pages.edit.unknown_error')
```

Comment:
Is this a native object now?

Reply:
In `app/assets/javascripts/translations.js` (via the i18n-js gem)

Reply:
thanks